### PR TITLE
Added new tutorial: Install Java Runtime Environment

### PR DESCRIPTION
**Pull request by aden-chrome**

### CODE REVIEW

File: tutorials/desktop/install-jre/install-jre.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+id: install-jre
+summary: Installation of Java Runtime Environment (JRE) on Ubuntu
+categories: desktop
+tags: tutorial,installation,ubuntu,desktop,java,jre
+difficulty: 1
+status: Published
+published: 2017-12-02
+author: Aden Padilla <adenpadilla@gmail.com>
+
```
Comment (caldav) [inline tutorials/desktop/install-jre/install-jre.md]: Missing `feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues`

File: tutorials/desktop/install-jre/install-jre.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+---
+
+# Install Java Runtime Environment (JRE)
+
+## Overview
+Duration: 1:00
+
+Java Runtime Environment (JRE) is required to run Java programs. There are several versions of Java and the ones we can install on our Ubuntu machines are OpenJRE and Oracle JRE. Both of them are similar and should not have any functional difference but some prefer OpenJRE over Oracle JRE as it is easier to install because it requires no additional repositories to be added. However, there are cases where you might need the other one.
```
Comment (caldav) [inline tutorials/desktop/install-jre/install-jre.md]: Suggestion for this paragraph: 

"The Java Runtime Environment (JRE) is required to run Java programs. While there are multiple JRE variants available, the two most popular ones on Ubuntu are OpenJRE and Oracle JRE. Using one over the other should not create any functional difference in most applications, however, some prefer OpenJRE over Oracle JRE as it doesn't contain closed-source components and is maintained as part of the Ubuntu archive, with easier installation and upgrades."

File: tutorials/desktop/install-jre/install-jre.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+In this guide, we'll be going through the installation of both of them. Of course, you generally only need one so pick one that best suites your need.
```
Comment (caldav) [inline tutorials/desktop/install-jre/install-jre.md]: so pick one -> to pick the one
suites -> suits
need -> needs

File: tutorials/desktop/install-jre/install-jre.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+```bash
+sudo apt install openjdk-8-jre
+```
+
+We can check if OpenJRE was properly installed by running:
+```bash
+java -version
+```
+
+The following should be its output:
```
Comment (caldav) [inline tutorials/desktop/install-jre/install-jre.md]: (simplify) "It should output the following:"

File: tutorials/desktop/install-jre/install-jre.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+openjdk version "1.8.0_151"
+OpenJDK Runtime Environment (build 1.8.0_151-8u151-b12-0ubuntu0.16.04.2-b12)
+OpenJDK 64-Bit Server VM (build 25.151-b12, mixed mode)
+
+```
+And that's it!
+
+In the next step we'll install Oracle JRE.
+
+## Install Oracle JRE
```
Comment (caldav) [inline tutorials/desktop/install-jre/install-jre.md]: Your previous title is "Installing OpenJRE" you should use "Installing" here as well.

File: tutorials/desktop/install-jre/install-jre.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+Duration: 2:00
+
+Oracle doesn't provide an official install package for Ubuntu but luckily **WebUp8Team** created a script that installs Oracle JRE for us.
```
Comment (caldav) [inline tutorials/desktop/install-jre/install-jre.md]: Try not to rely on third party scripts, you should look into Oracle doc https://docs.oracle.com/javase/9/install/installation-jdk-and-jre-linux-platforms.htm#JSJIG-GUID-D54A6F28-A305-4249-A5BF-835B25644C26 .

File: tutorials/desktop/install-jre/install-jre.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+```bash
+sudo apt update
+```
+Then finally:
+```bash
+sudo apt install oracle-java8-installer
+```
+After agreeing to the install prompt, we'll be greeted with the following screen:
+![prompt1](images/notiagree.png)
+Read their license terms over at the link provided then press enter.
```
Comment (caldav) [inline tutorials/desktop/install-jre/install-jre.md]: **,** then press enter.

File: tutorials/desktop/install-jre/install-jre.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+Another screen will pop up asking if you agree to their terms.
+![prompt2](images/agreeprompt.png)
+If you do, select yes and the installation will proceed. All we have to do now is sit back and let the script do the rest of the work for us.
+
+When it's done, we can check if it was set up correctly by typing:
```
Comment (caldav) [inline tutorials/desktop/install-jre/install-jre.md]: ...we can check the installation by typing:

File: tutorials/desktop/install-jre/install-jre.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+java version "1.8.0_151"
+Java(TM) SE Runtime Environment (build 1.8.0_151-b12)
+Java HotSpot(TM) 64-Bit Server VM (build 25.151-b12, mixed mode)
+```
+
+And that concludes our tutorial. But don't let that stop you, there's a link below if you want to learn more!
+
+positive
+: **Read more:**
+https://help.ubuntu.com/community/Java#Installation_of_Java_Runtime_Environment
```
Comment (caldav) [inline tutorials/desktop/install-jre/install-jre.md]: The tutorial engine is not as smart as GitHub, use proper Markdown for links :) `[https://help.ubuntu.com/community/Java#Installation_of_Java_Runtime_Environment](https://help.ubuntu.com/community/Java#Installation_of_Java_Runtime_Environment)`

**Comment by aden-chrome** [comment]
@caldav I've made some changes to the tutorial based on your comments. Sorry this took so long, I've been a little busy lately. Let me know if there's anything else I need to change. Thanks!

**Pull request merged**